### ISSUE TITLE
[MLv2] [Bug] Missing schemas for cumulative sum (`:cum-sum`) and cumulative count (`:cum-count`)aggregations

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#29942]
```clj
Don't know how to determine the type of 
[:cum-sum #:lib{:uuid "8abfffc6-0531-47e4-bb99-7837c92b1436"} [:field #:lib{:uuid "f2b32892-bdf7-4725-9d96-dd7931abe072"} 133245]]
```

**Comment by camsaul** [comment]
The fix for this should remove the code to skip tests for queries matching this issue in `metabase.query-processor-test.test-mlv2`

**Comment by metamben** [comment]
Resolved by #30836.
